src/modules/RelocationServer.rb

```diff
@@ -261,7 +261,7 @@ def ReadLibvirtServices
       end
 
       ports = SuSEFirewallServices.GetNeededTCPPorts(
```

Comment:
SuSEFirewall should be replaced by firewalld. I already checked that the service definition is already provided by the libvirt package, the only question if there is some firewalld default service that already provides that definition and if not if the definition should be added in upstream or to our own package.